lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -35,20 +38,21 @@ def check_output2(*popenargs, **kwargs):
     if 'input' in kwargs:
         if 'stdin' in kwargs:
             raise ValueError('stdin and input arguments may not both be used.')
-        inputdata = kwargs['input']
+        inputdata = to_bytes(kwargs['input'])
         del kwargs['input']
         kwargs['stdin'] = subprocess.PIPE
     else:
         inputdata = None
     process = subprocess.Popen(*popenargs, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs)
     try:
         out, err = process.communicate(inputdata)
+        out, err = to_native(out), to_native(err)
```

Comment:
rather than transforming these here (which makes the output different from check_output from the python stdlib) I would make other things in this function deal with them as bytes.  So this would be:
``` python
b_out, b_err = process.commincate(inputdata)
```